lib/assets/stylesheets/tokens/variables/_colors.scss

```diff
@@ -4,7 +4,7 @@ $colors-greyscale: (
   "fog": #979899,
   "elephant": #cacbcc,
   "noise": #f2f3f5,
-  "snow": #ffffff
+  "snow": #ffffff,
```

Comment:
these commas are added on save. Can't really save without it 0_o

Reply:
Sounds like a linter issue.  We'll fix it eventually.